python/btcspv/validate_spv.py

```diff
@@ -0,0 +1,170 @@
+from riemann import tx
+
+from riemann import utils as rutils
+
+from btcspv.types import RelayHeader, SPVProof
+
+from btcspv.utils import verify_proof
```

Comment:
James's rules for python imports:

1. separate blocks of imports by a blank line
1. Top block is common libraries. including standard lib and things like `aiohttp`
1. Any number of middle blocks, conceptually group (e.g. this block is bitcoin things, this block is ether things)
1. Final block is type imports
1. Within a block, sort lines by line length
1. import types by name, when using functions and non-type attributes import the module and reference by name
    1. yes: `from btcspv.types import SPVProof`
    1. yes: `from btcspv import utils`
    1. no: `from btcspv.utils import verify_proof`

Reply:
Example:
```
import aiohttp  # don't need this in this file, just an example of a top block

from riemann import tx
from riemann import utils as rutils

from btcspv import utils

from btcspv.types import RelayHeader, SPVProof
```

Reply:
So, this? ```from riemann import tx
from btcspv import utils
from riemann import utils as rutils

from btcspv.types import RelayHeader, SPVProof```

Reply:
Oh, never mind, you gave example in next comment